cylc/flow/cfgspec/globalcfg.py

```diff
@@ -971,59 +972,84 @@
                 ssh
                    Use non-interactive ssh for task communications
             ''')
-            # TODO ensure that it is possible to over-ride the following three
-            # settings in workflow config.
-            Conf('submission polling intervals', VDR.V_INTERVAL_LIST, desc=f'''
+            Conf('submission polling intervals',
+                 VDR.V_INTERVAL_LIST,
+                 [DurationFloat(900)],
+                 desc='''
                 List of intervals at which to poll status of job submission.
 
-                {MOVEDFROMJOB}
-
                 Cylc can poll submitted jobs to catch problems that
                 prevent the submitted job from executing at all, such as
-                deletion from an external job runner queue. Routine
-                polling is done only for the polling ``task communication
-                method`` unless workflow-specific polling is configured in
-                the workflow configuration. A list of interval values can be
-                specified as for execution polling but a single value
-                is probably sufficient for job submission polling.
+                deletion from an external job runner queue. A list of interval
+                values can be specified, with the last value used repeatedly
+                until the task starts running.
+                Multipliers can be used as shorthand as in the example below.
 
                 Example::
 
-                   5*PT1M, 10*PT5M
+                   5*PT2M, PT5M
+
+                Note that if the polling
+                :cylc:conf:`global.cylc[platforms][<platform name>]
+                communication method`
+                is used then Cylc relies on the polling to detect when a task
+                starts running so you are likely to want to configure more
+                frequent polling.
             ''')
-            Conf('submission retry delays', VDR.V_INTERVAL_LIST, None, desc=f'''
-            {MOVEDFROMJOB}
+            Conf('submission retry delays', VDR.V_INTERVAL_LIST, None,
+                 desc='''
+                Cylc can automate resubmission of a task job which has not
+                been submitted sucessfully.
+
+                Execution retry delays are a list of ISO 8601
+                durations/intervals which tell Cylc how long to wait before
+                resubmitting a failed job.
```

Comment:
```suggestion
                This setting is a list of ISO 8601
                durations/intervals, for how long to wait before
                resubmitting failed jobs.
```

Reply:
```suggestion
                A list of ISO 8601 durations/intervals which define when
                the scheduler will resubmit failed jobs.
```

Reply:
Yes that's fine.

  I have a stylistic dislike of repeating the subject immediately after it was already stated (when, I think, the subject is clear from the context).

Some back up from an academic English style guide:

https://owl.purdue.edu/owl/general_writing/academic_writing/sentence_variety/for_repeated_subjects_or_topics.html

(As a general rule, I also don't like passive voice in technical writing. It tends to be unnecessarily verbose and sounds kind of flaccid).

Reply:
> As a general rule, I also don't like passive voice in technical writing. It tends to be unnecessarily verbose and sounds flaccid. (This is to explain many of my doc PR review suggestions, not yours in particular!)

Agreed, even if I am a culprit.

Reply:
N.B - Talking to Dave made this actually about submission and _not_ execution retries.